content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -107,89 +99,123 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
+Invoke-Expression `$kubeletCommandLine
+"@ | Set-Content -Path c:\k\Start-kubelet.ps1
+
+c:\k\nssm.exe install kubelet Powershell -ExecutionPolicy Bypass -NoProfile c:\k\Start-kubelet.ps1
+c:\k\nssm.exe set Kubelet AppStdout C:\k\kubelet.log
+c:\k\nssm.exe set Kubelet AppStderr C:\k\kubelet.err.log 
+```
 
-```powershell
-# Example
-.\Install-Containerd.ps1 -ContainerDVersion 1.4.1
+We need to open a firewall port:
+
+```
+New-NetFirewallRule -Name kubelet -DisplayName 'kubelet' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 10250
 ```
 
-If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, specify the name with `-netAdapterName`.
+Verify kubelet is installed with `Get-service kubelet`.  Kubeadm will start kubelet during joining the node. 
 
-```powershell
-# Example
-.\Install-Containerd.ps1 -netAdapterName "Ethernet0 2"
+```
+Get-Service kubelet
+Status   Name     DisplayName
+------   ----     -----------
+Stopped  kubelet  kubelet
 ```
 
-{{< /note >}}
+#### Install kubeadm
 
-#### Install wins, kubelet, and kubeadm
+Start a Powershell session, set `$KubernetesVersion` to the desired version (ex: `$KubernetesVersion="v1.24.0"`), and then run the following commands:
 
-```PowerShell
-curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/kubeadm/scripts/PrepareNode.ps1
-.\PrepareNode.ps1 -KubernetesVersion {{< param "fullversion" >}} -ContainerRuntime containerD
 ```
-
-[Install `crictl` from the cri-tools package](https://github.com/kubernetes-sigs/cri-tools)
-which is required so that kubeadm can talk to the CRI endpoint.
+curl.exe -L https://dl.k8s.io/$KubernetesVersion/bin/windows/amd64/kubeadm.exe -o c:\k\kubeadm.exe
+```
 
 #### Run `kubeadm` to join the node
 
-Use the command that was given to you when you ran `kubeadm init` on a control plane host.
+Use the command that was given to you when you ran `kubeadm init` on a control plane host. You will need to add  `--cri-socket "npipe:////./pipe/containerd-containerd"` to tell kubeadm to use the correct containerd pipe.
+
+The command will look like:
+
+```
+ ./kubeadm.exe join 10.240.0.10:6443 --token <your token> --discovery-token-ca-cert-hash sha256:<your hash> --cri-socket "npipe:////./pipe/containerd-containerd" 
+```
+
 If you no longer have this command, or the token has expired, you can run `kubeadm token create --print-join-command`
 (on a control plane host) to generate a new token and join command.
 
+#### Install Flannel CNI DaemonSet
+
+We will leverage host-process containers to run flannel as a DaemonSet:
+
+```
+kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay-hpc.yml
+```
+
+#### Install kube-proxy DaemonSet
+
+We will leverage host-process containers to run kube-proxy as a DaemonSet.  This can be run from any Linux machine that has kubectl installed with its context configured to your new cluster.
```

Comment:
- Please [address the reader as you](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you).
- Setting up networking is not specific to containerd, so this advice seems to be in the wrong place.
- Can you run `kubectl` on Windows, or does it require Linux?